std/variant.d

```diff
@@ -530,6 +530,8 @@ private:
         return 0;
     }
 
+    enum doUnittest = is(VariantN == Variant);
```

Comment:
btw his kind of peephole access during code review makes the individual tag more convenient than labels